aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -49,39 +49,32 @@ module aptos_framework::account {
 
     const MAX_U64: u128 = 18446744073709551615;
 
-    /// Account already existed
-    const EACCOUNT: u64 = 0;
-    /// Sequence number exceeded the maximum value for a u64
-    const ESEQUENCE_NUMBER_TOO_BIG: u64 = 1;
-    /// The address provided didn't match the `aptos_framework` address.
-    const ENOT_APTOS_FRAMEWORK: u64 = 2;
+    /// Account already existed.
+    const EACCOUNT_ALREADY_EXISTS: u64 = 1;
+    /// Sequence number exceeded the maximum value for a u64.
+    const ESEQUENCE_NUMBER_TOO_BIG: u64 = 2;
     /// The provided authentication had an invalid length
     const EMALFORMED_AUTHENTICATION_KEY: u64 = 3;
-
-    const ECANNOT_CREATE_AT_VM_RESERVED: u64 = 4;
-    const EGAS: u64 = 5;
-    const ECANNOT_CREATE_AT_CORE_CODE: u64 = 6;
-    const EADDR_NOT_MATCH_PREIMAGE: u64 = 7;
-    const EWRITESET_NOT_ALLOWED: u64 = 8;
-    const EMULTI_AGENT_NOT_SUPPORTED: u64 = 9;
-    const EMODULE_NOT_ALLOWED: u64 = 10;
-    const ESCRIPT_NOT_ALLOWED: u64 = 11;
+    /// Cannot create an account because address has been reserved by the system.
```

Comment:
`Cannot create account because address is reserved`

Reply:
Updated